frontend/src/App.vue

```diff
@@ -435,4 +435,12 @@ span.sc {
   white-space: nowrap;
 }
 
+.break-word {
+  word-break: break-word;
+  -webkit-hyphens: auto;
+  -moz-hyphens: auto;
+  -ms-hyphens: auto;
```

Comment:
These lines didn't seem to be picked up by the target browsers (Firefox, Edge), could you have another look?
```suggestion
```